app/assets/javascripts/components/accountPanel/accountPanel.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import { actions as appActions } from 'redux/modules/app.js';
+
+// Components
+import Sidebar from 'components/accountPanel/sidebar.jsx';
+
+class AccountPanel extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  createTabIfNotCreated() {
+    if (this.shouldCreateTab()) {
+      const tab = {
+        id: 'account',
+        label: 'Account',
+        link: '/dashboard/account',
+        closeable: true,
+      };
+
+      this.props.dispatch(appActions.addTab(tab));
+    };
+  }
+
+  findAccountTab() {
+    const { tabs } = this.props;
+    return (_.find(tabs, { id: 'account' }));
+  }
+
+  shouldCreateTab() {
+    return (this.findAccountTab() ? false : true);
+  }
+
+  componentWillMount() {
+    this.createTabIfNotCreated();
+  }
+
+  cloneChildren() {
+    const { brand, subscription } = this.props;
+    return React.Children.map(this.props.children, (child) => {
+      return React.cloneElement(child, { brand, subscription });
```

Comment:
Shouldn't need to do this. I'd connect in the subroute components instead of passing it through here.